### DISCUSSION TITLE
Disabling multishipping or forcing carrier selection for every linked order

### DISCUSSION THREAD

**Discussion by joseharitza** [PrestaShop/PrestaShop#34289] [Ideas]
### The problem

As Prestashop takes the decision of witch carrier should apply on linked orders it is usually the cheapest one (usually the free one used for the Pick up option)

When this option it is not realistic for the client it becomes frustrating to explain clients that they have paid less than they should.

### The solution

The simple solution would be to let as an option to disable Linked orders.

### Alternatives

The advanced alternative would be to force the client to chose the carrier for every of the linked orders.

### Do you plan to work on this subject?

- [X] I'm willing to contribute a [formal specification](https://github.com/PrestaShop/prestashop-specs).
- [X] I'm willing to provide any wireframes or design assets required for this feature.
- [X] I'm willing to submit a Pull Request that implements this feature.
- [X] I'm willing to help verify that the implemented feature works as intended and produces no unintended side effects.

**Comment by kpodemski** [comment]
Hello @joseharitza 

The multi-shipping feature will be removed in version 9 of PrestaShop, thanks to @Hlavtox, who removed its remains. This feature didn't work correctly, and it's been decided to remove it.

**Reply by Hlavtox** [reply]
@kpodemski I removed the multi-address part. ;-) Orders will still be split if the order contains uncompatible carreirs, I think we should keep this functionality.

**Reply by joseharitza** [reply]
Hello @Hlavtox

Will orders split even if there are country restrictions in carriers in Prestashop 9? I have noticed that with multi-shipping some orders are passed even if delivery address is for a country (or zone) not enabled on the carrier that is finally taken by Prestashop.

I had no idea that it was possible to pass orders with multi address choice.

Thx
